src/FileHandler.cpp

```diff
@@ -115,9 +115,9 @@ std::string FileHandler::getFileResource(HttpRequest *request, std::ios_base::op
 	return body;
 }
 
-std::string FileHandler::getFileContent(std::string path)
```

Comment:
this name should be changed to something like get with absolut path or get cgi path or smt that tells us the fundamental difference with the other get file resource function.. also, this particular function was originally meant to get the internal files like for example the configuration file etc where we dont need to use the relative path so Id feel mor comfortable creating a new function rather than using this one and changing its behavior... although I did look that this one is not used anywhere I guess at the moment, but anyway the name should be changed

Reply:
yeah, I would not have modified it, if someone else was using it. I can change the name tho.